### PR TITLE
FSCRAWLER-151 Add owner & group of a file to ES index

### PR DESCRIPTION
**Pull request by sgoeschl**
* Adding a "attribute" class containing "owner" & "group" to the import and ES mapping
* Added integration test
* Updated the documentation

Please note that Windows has no concept of "group"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -644,25 +672,28 @@ PUT docs/doc/_mapping
...
+| `file.filename`                        | Original file name                          | `"mydocument.pdf"`                          |
+| `file.url`                             | Original file url                           | `"file://tmp/mydir/otherdir/mydocument.pdf"`|
+| `path.encoded`                         | MD5 encoded file path (for internal use)    | `"112aed83738239dbfe4485f024cd4ce1"`        |
+| `path.virtual`                         | Relative path from root path                | `"mydir/otherdir"`                          |
+| `path.root`                            | MD5 encoded root path (for internal use)    | `"112aed83738239dbfe4485f024cd4ce1"`        |
+| `path.real`                            | Actual real path name                       | `"/tmp/mydir/otherdir/mydocument.pdf"`      |
+| `attributes.owner`                     | Owner name                                  | `"david"`                                   |
+| `attributes.group`                     | Group name                                  | `"stuff"`                                   |
```
Comment (dadoonet) [inline README.md]: s/stuff/staff ?

**Comment by dadoonet** [comment]
I left a small comment after a quick review.
Could you rebase on latest master branch which now contains your commit for #149?

Looks really cool. Great stuff!

I'll try to test it and merge it soonish.

**Comment by sgoeschl** [comment]
Done (hopefully correct) :-)

**Comment by dadoonet** [comment]
Rebase would have been better but I'll do it. No worries

**Comment by sgoeschl** [comment]
Not enough Git foo :-(

**Comment by dadoonet** [comment]
I started to merge your PR.

I'm going to add also two lines in `FileAbstractorSSH`:

```java
        model.owner = Integer.toString(file.getAttrs().getUId());
        model.group = Integer.toString(file.getAttrs().getGId());
```

So we could get also this information from SSH as well.

I will also make this new structure as optional.

**Comment by dadoonet** [comment]
I merged it with 85a8c4f68a00df09aa83e3573f16c92bb52c3511.

Thanks a lot!

**Pull request closed without merging**